### ISSUE TITLE
[Multi Project Selection] Checkbox Tooltip Not Clear

### ISSUE THREAD

**Issue by souredoutlook** [getsentry/sentry#70516]
### Problem Statement

A user wrote in with the misconception that the CTA pointing folks to "upgrade for Multi Project selection" at first glance appears as though Replays for that project were gated behind Business plan.

From the user:

> why would I have a checkbox for a feature that I can't use?
If I see check box as a user I assume it's to be clicked, am I supposed to guess that label is clickable when the rest of the world does not mix click able labels and check boxes while making them serve different purposes.

### Solution Brainstorm

From the user:

> If multiproject is not enabled just don't show the checkbox. It's just straight up bad UX to have controls for features that are not there and usability of components clashing with common practices of the rest of the internet. 

### Product Area

Unknown

**Comment by jangjodi** [comment]
I agree that it's not clear that the checkbox is only used for multi project select, where as the label is used to change the selected projects. A potential solution would be to:
- Either remove the checkbox or allow clicking the checkbox to change the project instead of giving the "upgrade" CTA
- Move the message for upgrading to allow for multi project selection to the bottom
<img width="421" alt="Screenshot 2024-05-14 at 9 12 19 AM" src="https://github.com/getsentry/sentry/assets/116035587/0a6d3a0e-a7a7-4d24-94db-3e192003c425">
